test/output/netcdf_output.jl

```diff
@@ -142,7 +153,22 @@ end
     mslp = ds["mslp"].var[:, :, end]    # variable at last time step `.var` to read the raw data ignoring any mask
     
     # should be within ~800 to ~1200hPa
-    @test all(0.8 .< mslp/pres_ref .< 1.2)
+    @test all(0.8 .< mslp./pres_ref .< 1.2)
+
+    ## test u10, v10 existence
+    @test haskey(ds, "u")
+    @test haskey(ds, "u10")
+    @test haskey(ds, "v")
+    @test haskey(ds, "v10")
+
+    # and 10m values cannot exceed lowermost layer of u,v value
+    @test maximum(abs.(ds["u10"].var[:, :, end])) < maximum(abs.(ds["u"].var[:, :, end, end]))
+    @test maximum(abs.(ds["v10"].var[:, :, end])) < maximum(abs.(ds["u"].var[:, :, end, end]))
```

Comment:
you used `max(array...)` I changed this to `maximum` which is the correct way of doing this in Julia as it's non-allocating. Splatting out arrays is generally not good practice unfortunately